en,zh: add pause deployment docs of tidb cluster (#906)
pingcap/docs-tidb-operator: en/pause-deployment-of-tidbcluster.md

---
title: Pause Deployment of TiDB Cluster in Kubernetes

> summary: Interduce how to pause deployment of TiDB Cluster in Kubernetes

Reviewer: ```suggestion
summary: Introduce how to pause a TiDB Cluster in Kubernetes
```

Author: fixed
